README.md

```diff
@@ -86,22 +85,19 @@ Martin provides [TileJSON](https://github.com/mapbox/tilejson-spec) endpoint for
 
 When started, martin will go through all spatial tables and functions with an appropriate signature in the database. These tables and functions will be available as the HTTP endpoints, which you can use to query Mapbox vector tiles.
 
-| Method | URL                                                                              | Description                                             |
-|--------|----------------------------------------------------------------------------------|---------------------------------------------------------|
-| `GET`  | `/index.json`                                                                    | [Table Sources List](#table-sources-list)               |
-| `GET`  | `/{schema_name}.{table_name}.json`                                               | [Table Source TileJSON](#table-source-tilejson)         |
-| `GET`  | `/{schema_name}.{table_name}/{z}/{x}/{y}.pbf`                                    | [Table Source Tiles](#table-source-tiles)               |
-| `GET`  | `/{schema_name1}.{table_name1},...,{schema_nameN}.{table_nameN}.json`            | [Composite Source TileJSON](#composite-source-tilejson) |
-| `GET`  | `/{schema_name1}.{table_name1},...,{schema_nameN}.{table_nameN}/{z}/{x}/{y}.pbf` | [Composite Source Tiles](#composite-source-tiles)       |
-| `GET`  | `/rpc/index.json`                                                                | [Function Sources List](#function-sources-list)         |
-| `GET`  | `/rpc/{schema_name}.{function_name}.json`                                        | [Function Source TileJSON](#function-source-tilejson)   |
-| `GET`  | `/rpc/{schema_name}.{function_name}/{z}/{x}/{y}.pbf`                             | [Function Source Tiles](#function-source-tiles)         |
-| `GET`  | `/healthz`                                                                       | Martin server health check: returns `200 OK`            |
+| Method | URL                                                    | Description                                             |
+|--------|--------------------------------------------------------|---------------------------------------------------------|
+| `GET`  | `/`                                                    | [List of all sources](#source-list)                     |
```

Comment:
Let's use `/` for the future user interface and `/catalog` for the list of all sources. We might want to add a list of reserved keywords for that, the same way as with `/healthz`

Reply:
thanks, fixed with reserved keywords, and added a `/catalog`